server/api/controllers/liveSermon/create-live-sermon.js

```diff
@@ -0,0 +1,103 @@
+module.exports = {
+  friendlyName: 'Create Live Sermon',
+
+  description: 'Create Live Sermon',
+
+  inputs: {
+    title: {
+      type: 'string',
+      required: true,
+    },
+    sermonNotes: {
+      type: 'string',
+    },
+    streamLink: {
+      type: 'string',
+      required: true,
+    },
+    speaker: {
+      type: 'string',
+      required: true,
+    },
+    sermonSeries: {
+      type: 'string',
+    },
+    sermonDate: {
+      type: 'string',
+      required: true,
+    },
+    sermonPassage: {
+      type: 'string',
+    },
+    sermonDescription: {
+      type: 'string',
+    },
+    isPublished: {
+      type: 'boolean',
+    },
+    lastUpdatedBy: {
```

Comment:
i would remove this. and just always assign lastUpdatedBy in your function implementation